packages/data-provider/src/core/provider.ts

```diff
@@ -50,13 +52,15 @@ const provider = (configData: AppConfig) => {
 			dictionary: dictionaryRouter(baseDeps),
 			submission: submissionRouter(baseDeps),
 			submittedData: submittedDataRouter(baseDeps),
+			validator: validationRouter(baseDeps),
 		},
 		controllers: {
 			audit: auditController(baseDeps),
 			category: categoryController(baseDeps),
 			dictionary: dictionaryController(baseDeps),
 			submission: submissionController(baseDeps),
 			submittedData: submittedDataController(baseDeps),
+			validator: validationService(baseDeps)
```

Comment:
`controller` should have a `validationController` and `services` a `validationService`